components/widgets/resources/src/main/resources/resources/uicomponents/widgets/exportData.css

```diff
@@ -125,3 +125,9 @@
 .filter-match-count {
   margin-left: 0.5em
 }
+input.suggestUsersAndGroups {
+  background-image: url(/resources/icons/datamodel/users.png);
```

Comment:
This doesn't work when PhenoTips is not deployed as the root webapp. Use `$xwiki.getSkinFile('icons/datamodel/users.png')`